src/mumble/MainWindow.cpp

```diff
@@ -387,6 +387,10 @@ void MainWindow::setupGui()  {
 #endif
 }
 
+void MainWindow::updateWindowTitle() {
+	setWindowTitle((g.s.bMinimalView ? tr("Mumble - Minimal View -- %1") : tr("Mumble -- %1")).arg(QLatin1String(MUMBLE_RELEASE)));
```

Comment:
I would use an if block here.  It seems to me we might add more titles in the future, so er might as well future proof it a bit?